content/sphinx.md

```diff
@@ -323,42 +325,36 @@ the list of extensions in `conf.py`:
 extensions = ['sphinx.ext.mathjax']
 ```
 
+**Add here: https://myst-parser.readthedocs.io/en/latest/syntax/optional.html#syntax-amsmath**
+
 The following shows how to inline mathematics within a text:
 
-```rst
-This is an inline equation embedded :math:`a^2 + b^2 = c^2` in text.
+```md
+This is an inline equation embedded {math}`a^2 + b^2 = c^2` in text.
 ```
 
-An equation and equation array:
-```rst
-.. math::
-  :label: myequation
+An equation:
 
-  a^2 + b^2 = c^2
+:math:`a^2 + b^2 = c^2`
 
+or:
 .. math::
-  :label: myarray
-
-  \begin{eqnarray}
-    x^2 & : x < 0 \\
-    x^3 & : x \ge 0 \\
-  \end{eqnarray}
-```
+`a^2 + b^2 = c^2`
 
-These equations can then be referenced using ``:eq:`myequation` `` and ``:eq:`myarray` ``.
 ````
 
 ---
 
 ### Where to find more
 
-- For more RST functionality, see the [Sphinx documentation](http://www.sphinx-doc.org/en/stable/rest.html)
-  and the [quick-reference](http://docutils.sourceforge.net/docs/user/rst/quickref.html).
-- For Sphinx additions to standard RST, see [Sphinx Markup Constructs](http://www.sphinx-doc.org/en/1.7/markup/index.html).
+- For more Markdown functionality, see the [Mardown guide](https://www.markdownguide.org/basic-syntax/).
+- For Sphinx additions, see [Sphinx Markup Constructs](http://www.sphinx-doc.org/en/1.7/markup/index.html).
```

Comment:
Maybe we should clarify MD vs RSE vs MyST more.  But let's merge and I can try this on a second pass?

In particular linking to any MD guide except the MyST one will exclude all of the important Sphinx features - I guess this is a strategic problem here, the way the term "markdown" will become very confusing to learners.